frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -948,6 +948,11 @@ pub mod pallet {
 				compute: ElectionCompute::Emergency,
 			};
 
+			Self::deposit_event(Event::SolutionStored {
+				election_compute: ElectionCompute::Emergency,
+				prev_ejected: Self::queued_solution().is_some(),
```

Comment:
This will increase weight to two reads I think.  
Sorry for always pointing these out :sweat_smile:

Reply:
done.